<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>jshark::forms::MainWidget</class>
 <widget class="QWidget" name="jshark::forms::MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>JShark</string>
  </property>
  <widget class="QComboBox" name="deviceComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>191</width>
     <height>16</height>
    </size>
   </property>
   <property name="text">
    <string>Select Output Device</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="outputTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>381</width>
     <height>171</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="selectOutputFileButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Output File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="installButton">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>70</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Install/Update Settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeJSharkButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove JShark</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
